frontend/src/app/core/auth/local-auth/user-management/forms/import-csv-dialog/import-csv-dialog.component.html

```diff
@@ -10,27 +10,27 @@
     <mat-icon matSuffix>folder</mat-icon>
   </mat-form-field>
   <div style="max-height: 600px; overflow-y: auto">
-    <table [dataSource]="rowData" mat-table matSort>
```

Comment:
Hier scheint das table-tag verloren zu gehen. Bitte schaue dir die Änderungen in dieser Datei nochmal genauer an